spec/hosts/roles/hexrot_spec.rb

```diff
@@ -32,16 +32,85 @@
           it { is_expected.to contain_class('docker') }
 
           %w[
+            anaconda
+            mate
             profile::core::common
             profile::core::debugutils
```

Comment:
Replace with `include_examples 'debugutils'`

Reply:
replaced and removed 👍🏻